release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -120,6 +120,36 @@ As an administrator, you can specify multiple failure domains for your {product-
 
 [id="ocp-4-13-web-console"]
 === Web console
+[id="ocp-4-13-developer-perspective"]
+==== Developer Perspective
+
+With this release, there are several updates to the *Developer* perspective of the web console. You can perform the following actions:
+
+* Create a *Serverless Function* through the *Import from Git* flow.
+* Create a *Serverless Function* through *Create Serverless Function* flow available on *Add page*.
+* The *Import from Git* workflow now supports *pipeline-as-code*.
+* See which pods are receiving traffic in the:
+** Side pane of the *Topology* view
+** Pod *Details* view
+** *Pods* list view
+* Customize the timeout period or provide your own image when instantiating a *Web Terminal*.
+* Set default resources as an administrator to be pre-pinned in the *Developer* perspective navigation for all the users.
+
+[id="ocp-4-13-pipelines-page-improvements"]
+===== Pipelines page improvements
+In {product-title} 4.13, you can see the following navigation improvements on the *Pipelines* page:
+
+* The previously selected tab is visible when returning to the *Pipelines* page.
+* The default tab for *Repository details* page is now *PipelinesRuns*, but when you are following the *Create Git Repository* flow, the default tab is *Details*.
+
+[id="ocp-4-13-helm-page-improvements"]
+===== Helm page improvements
+In {product-title} 4.13, you can do the following from the *Helm* page:
+
+* New terminology: Create and delete Helm releases instead of installing and uninstalling Helm charts.
+* Create and delete helm releases asynchronously and not wait for these actions to complete before performing the next task in the web console.
+* Helm release list now includes a *Status* column.
```

Comment:
```suggestion
* The Helm release list now includes a *Status* column.
```
- Add definite article.
- Should `Helm` use an attribute?

Reply:
Made the first change and skipping the second suggestion.